deploy/apecloud-mysql-cluster/values.yaml

```diff
@@ -2,66 +2,58 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+## @param version MySQL cluster version
+##
+version: ac-mysql-8.0.30
 
-terminationPolicy: Delete
+## @param mode MySQL cluster topology mode, standalone, replication or raftGroup
+##
+mode: standalone
 
-clusterVersionOverride: ""
-nameOverride: ""
-fullnameOverride: ""
+## @param replicas MySQL cluster replicas
+## for standalone mode, replicas can not be changed, it is always 1
+## for replication mode, replicas can not be changed, it is always 2
```

Comment:
I we allow 2 or more replicas for replication workloads

Reply:
Ok, modified.